llvm/lib/CodeGen/GlobalISel/CSEInfo.cpp

```diff
@@ -65,6 +65,12 @@ bool CSEConfigFull::shouldCSEOpc(unsigned Opc) {
   case TargetOpcode::G_BUILD_VECTOR:
   case TargetOpcode::G_BUILD_VECTOR_TRUNC:
   case TargetOpcode::G_SEXT_INREG:
+  case TargetOpcode::G_FADD:
+  case TargetOpcode::G_FSUB:
+  case TargetOpcode::G_FMUL:
+  case TargetOpcode::G_FDIV:
+  case TargetOpcode::G_FNEG:
+  case TargetOpcode::G_FABS:
```

Comment:
The various mins and maxes also

Reply:
question: should we fold `G_FMINNUM/G_FMANNUM`? the result of comparing 0 and -0 could be different.

Reply:
Yes. It also couldn't be. We're also fixing the definition to always have the correct ordered zero handling, so it would just be subject to the ordinary nsz flag handling